test/unit/mixin-behaviors.html

```diff
@@ -552,7 +551,13 @@
   });
 
   test('nested-behavior dedups', function() {
-    assert.equal(el.behaviors.length, 4);
+    assert.equal(el.behaviors.length, 2);
```

Comment:
This also seems like a regression to me. Are we sure users will not run into any breaking changes?

Reply:
It's not a regression since the test changed, but the behavior is weird and I think we should change it.

We didn't have a test where `Polymer.mixinBehaviors` was called on a legacy element with behaviors (since this is pretty exotic) and this test was just updated to do that.

When that happens, the behaviors list is set to the subclasses' list of behaviors. However, the "active" set of behaviors on the element is the de-duplicated flat list of all behaviors on anything in the prototype chain. Given that, it seems like `.behaviors` should be this de-duplicated flattened list of "active" behaviors. We can make that change and update the test.